src/observer.tsx

```diff
@@ -1,31 +1,31 @@
-/* eslint-disable react/display-name */
-import React, { Component, forwardRef, memo } from "react"
+import * as React from "react"
 import { observer as observerLite, Observer } from "mobx-react-lite"
 
 import { makeClassComponentObserver } from "./observerClass"
+import { IReactComponent } from "./index"
 
-const hasSymbol = typeof Symbol === "function" && Symbol.for
+const hasSymbol: false | ((key: string) => symbol) = typeof Symbol === "function" && Symbol.for
```

Comment:
Another example of useless explicit types. If you remove them and hover over `hasSymbol` you see that TS has inferred exactly the same thing. So it only adds noise to the code.
```suggestion
const hasSymbol = typeof Symbol === "function" && Symbol.for
```

Reply:
Yep.  Forgot that one.  Thanks!